### ISSUE TITLE
path in browser has a "?_k=qvx3sz" param

### ISSUE THREAD

**Issue by tonyisid** [rackt/react-router#1907]
I update react-router to v1.0.0.0-rc1, the path in browser like that:
http://0.0.0.0:8000/#/account/pAuth?_k=qvx3sz

It is a clean path without any param in react-router v0.13 or other version, wehn I use "Route name=....".

Can I get a clean path by config something?

**Comment by chentsulin** [comment]
see https://github.com/rackt/react-router/blob/master/docs/basics/Histories.md#what-is-that-_kckuvup-junk-in-the-url

**Comment by JoshAshby** [comment]
I just ran into this as well. Using the browser history isn't really an option for me, as my components are part of a rails app with mostly haml template where react is being used to simply add some dynamic functionality to the pages. As a result the urls aren't always the nicest to work with and the hash history makes a little more sense in my use case. I don't care about preserving the state for these things, I got around it by doing

```javascript
createHashHistory({ queryKey: false })
```

which is mentioned in the [history docs](http://rackt.github.io/history/stable/HashHistoryCaveats.html). Obviously there are downsides, but its fine enough for me so far.
